src/sql/workbench/contrib/editData/browser/gridActions.ts

```diff
@@ -30,6 +31,7 @@ export class GridActionProvider {
 
 	constructor(
 		protected _dataService: DataService,
+		protected _configurationService: IConfigurationService,
```

Comment:
Any reason you're not using the dependency injection for this?

Reply:
Just following previous practices (dataService)

Reply:
Data service isn't an actual service that can be injected so that's probably why that wasn't done there. For DI-able services we should use the DI annotations.